projects/Bank-project/README.md

```diff
@@ -0,0 +1,4 @@
+# Gringotts Wizarding Bank
```

Comment:
please add a readme format as mentioned in the issue #30

Reply:
> please add a readme format as mentioned in the issue #30

@fykaa  I've fixed it, could you review the changes to approve that they satisfy the PR instructions